src/paths/library/metadata/[ratingKey]/get-media-meta-data.yaml

```
@@ -297,6 +296,10 @@ get:
                                                           - $ref: "../../../../models/media/part/stream/hearing-impaired.yaml"
                                                           - $ref: "../../../../models/media/part/stream/dub.yaml"
                                                           - $ref: "../../../../models/media/part/stream/title.yaml"
+                                                          - type: object
+                                                            properties:
+                                                              StreamType:                                                           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)